matrix-meetings-widget/src/components/meetings/MeetingsPanel/MeetingsPanel.test.tsx

```diff
@@ -868,6 +868,30 @@ describe('<MeetingsPanel/>', () => {
       }),
     ).toBeInTheDocument();
   });
+  it('should not show breakout session message form if the user has no permission to send message to all rooms', async () => {
+    enableBreakoutSessionView();
+
+    widgetApi.mockSendStateEvent(
+      mockPowerLevelsEvent({
+        room_id: '!room-id',
+        content: {
+          events: { 'net.nordeck.meetings.sub_meetings.send_message': 101 },
+        },
+      }),
+    );
+
+    render(<MeetingsPanel />, { wrapper: Wrapper });
```

Comment:
Testing if a component is not rendered is tricky with asynchronous events (is it not displayed or _not yet_ displayed?)

You can use a similar setup as in the test following this one ("should hide the actions navigation section if the user has no permission to create breakout sessions"):

1. `enableBreakoutSessionView();`
2. `render(...)`
3. `const textbox = ...getByRole('textbox', ...)` <- make sure everything is loaded correctly
4. `widgetApi.mockSendStateEvent( ... )` <- now change the power level
5. `await waitFor(() => expect(textbox).not.toBeInTheDocument());` <- check if it is now hidden